controls/datagrid/search-as-you-type.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Search as You Type
+page_title: .NET MAUI DataGrid Documentation - Search as You Type
+description: "Learn more about the search as you type feature of the Telerik .NET MAUI DataGrid - how to show/hide the search panel, various configuration settings of the search functionality, methods and events related to searching."
+position: 3
+slug: datagrid-search-as-you-type
+---
+
+# .NET MAUI DataGrid Search as You Type
+
+Telerik .NET MAUI DataGrid provides the ability to search for specific data within its ItemsSource by using its built-in search funcionality. The default behavior is to search as you type, but it can also be performed when the end user finishes typing and presses Enter.
+
+Through the DataGrid's `SearchSettings` property you can control when to show the search panel as well as configure the way the search is performed, so it best suits the end users' needs.
```

Comment:
```suggestion
Through the DataGrid's `SearchSettings` property, you can control when to show the search panel as well as configure the way the search is performed so that it best suits the end users' needs.
```